src/unistd.rs

```diff
@@ -1295,19 +1504,21 @@ impl LinkatFlags {
 /// # References
 /// See also [linkat(2)](https://pubs.opengroup.org/onlinepubs/9699919799/functions/linkat.html)
 #[cfg(not(target_os = "redox"))] // RedoxFS does not support symlinks yet
```

Comment:
Not sure if hard link is supported by RedoxFS, but linkat() is for hard link rather than symlink